Solutions/PaloAltoPrismaCloud/Parsers/PaloAltoPrismaCloud.txt

```diff
@@ -0,0 +1,167 @@
+// Usage Instruction :
+// Paste below query in log analytics, click on Save button and select as Function from drop down by specifying function name and alias as PaloAltoPrismaCloud.
+// Function usually takes 10-15 minutes to activate. You can then use function alias from any other queries (e.g. PaloAltoPrismaCloud | take 10).
+// Reference : Using functions in Azure monitor log queries : https://docs.microsoft.com/azure/azure-monitor/log-query/functions
+let Audit_view = view () { 
+    PaloAltoPrismaCloudAudit_CL
+| where isnotempty(user_g)
```

Comment:
In sample data there is no Guid provided for user and resourceName, hence it is failing at user_g and resourceName_g  to save parser. I have updated some sample GUID in **PaloAltoPrismaCloudAudit_CL.json** of user and resourceName fields to solve this issue.

can you please check and let us know if there is any change.

Thanks,
Mallikarjun

Reply:
Fields resourceName and user can be _s and _g. I will update sample data. Also the line in parser
`| where isnotempty(user_g)`
is excessive and must be removed.
Could I fix it in a new PR as this PR is already merged?

Reply:
@v-maudan - sample data looks ok after your change. We only need to remove this line from parser:
`| where isnotempty(user_g)`

Reply:
@vu-socprime , Thanks for your confirmation, please raise new PR.

Reply:
@v-maudan PR created - https://github.com/Azure/Azure-Sentinel/pull/2116